services/core-web/src/components/Forms/projectSummaries/ProjectSummaryForm.js

```diff
@@ -0,0 +1,263 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import { Form } from "@ant-design/compatible";
+import "@ant-design/compatible/assets/index.css";
+import { Typography, Row, Col } from "antd";
+import CustomPropTypes from "@/customPropTypes";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+import DocumentTable from "@/components/common/DocumentTable";
+
+const propTypes = {
+  projectSummary: CustomPropTypes.projectSummary.isRequired,
+  initialValues: PropTypes.objectOf(PropTypes.any).isRequired,
+  projectSummaryDocumentTypesHash: PropTypes.objectOf(PropTypes.string).isRequired,
+  projectSummaryAuthorizationTypes: PropTypes.objectOf(PropTypes.any).isRequired,
+  projectSummaryPermitTypesHash: PropTypes.objectOf(PropTypes.string).isRequired,
+};
+
+const transformAuthorizationTypes = (types) => {
+  const transformedObject = {};
+  types.forEach((type) => {
```

Comment:
should this be let not const?

Reply:
Const works here because we are only re-assigning object properties, not the object itself. If I were to re-assign the whole object than you would be correct.